Library/Homebrew/extend/os/mac/requirements/macfuse_requirement.rb

```diff
@@ -3,30 +3,30 @@
 
 require "requirement"
 
-class OsxfuseRequirement < Requirement
+class MacfuseRequirement < Requirement
   extend T::Sig
 
   def initialize(tags = [])
-    odeprecated "depends_on :osxfuse"
+    odeprecated "depends_on :macfuse"
     super(tags)
   end
 
-  download "https://osxfuse.github.io/"
+  download "https://macfuse.github.io/"
 
-  satisfy(build_env: false) { self.class.binary_osxfuse_installed? }
+  satisfy(build_env: false) { self.class.binary_macfuse_installed? }
 
   sig { returns(T::Boolean) }
-  def self.binary_osxfuse_installed?
-    File.exist?("/usr/local/include/osxfuse/fuse.h") &&
-      !File.symlink?("/usr/local/include/osxfuse")
+  def self.binary_macfuse_installed?
+    File.exist?("/usr/local/include/macfuse/fuse.h") &&
+      !File.symlink?("/usr/local/include/macfuse")
```

Comment:
Suggest you check for both macfuse and osxfuse here.

Reply:
Makes sense. Now performs both checks and symlinks both to `/usr/local/include/macfuse`, does this make sense?

Reply:
You need an or `||` here, since right now this will only return the results of checking for `osxfuse`.